easybuild/easyblocks/t/tensorflow.py

```diff
@@ -851,6 +866,22 @@ def build_step(self):
             # and download it if needed
             self.target_opts.append('--config=mkl')
 
+        # Use the same configuration (i.e. environment) for compiling and using host tools
+        # This means that our action_envs are (almost) always passed
+        # Fully removed in Bazel 6.0 and limitted effect after at least 3.7 (see --host_action_env)
+        if bazel_version < '6.0.0':
+            self.target_opts.append('--distinct_host_configuration=false')
+
+        for key, value in action_env.items():
+            option = key
+            if value is not None:
+                option += "='%s'" % value
+
+            self.target_opts.append('--action_env=' + option)
+            if bazel_version >= '3.7.0':
+                # See https://github.com/bazelbuild/bazel/commit/a463d9095386b22c121d20957222dbb44caef7d4
```

Comment:
extend the comment a bit to explain why also using `--host_action_env`

it's fine to keep the reference to the `bazel` commit, but don't force people to visit that codebase ;)